### ISSUE TITLE
Causes Chunk Lag with Sodium(+indium)

### ISSUE THREAD

**Issue by Shockks** [jexline/exline-furniture#8]
i figured out that this furniture mod definitely lag chunks when placing/breaking blocks or opening doors etc in a chunk on fabric with Sodium+indium mods. Removing those 2 mods removed the chunk lag when using exlines furniture mod. the best item to test with is the bird bath or Adirondack chair they are extremely laggy with sodium, other stuff like chairs etc cause lag. some things are worse than others. Hope it can be fixed somehow.

**Comment by jexline** [comment]
I have reduced the amount of voxels used in many items in v2.0.2. I had noticed a lag when breaking the table before, now is gone. I have a updated many blocks including the bird bath and chairs to have only 1 or 2 voxel shapes for collision. Hope this helps reduce lag. I do not use Sodium, but have noticed some improvement since updating. Let me know if it makes a difference for you!
https://www.curseforge.com/minecraft/mc-mods/exlines-furniture/files/3606745
